extensions/pgx_pgmq/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pgx_pgmq"
-version = "0.0.3-alpha.1"
+version = "0.5.1"
```

Comment:
Are we going to tie these versions together? I'd expect the crates development to slow over time and the extension to pick up.

Reply:
Let's go with `0.0.4` in that case